docs/Contributing/eval-pull-request.rst

```diff
@@ -0,0 +1,79 @@
+Evaluating a Pull Request
+*************************
+
+This page mostly targets the Longturn Admins who will be asked to review (evaluate) a submitted Pull Request.
+It is the policy of the Longturn community that all (100%) Pull Requests have at least one reviewer complete
+an evaluation of a change and either approve or make suggestions for improvements before a merge into the
+master branch.
+
+This page assumes the user knows how to use :file:`git`, compile Freeciv21 and use GitHub.
+
+:strong:`Update Local Sandbox`
+
+.. code-block:: rst
+
+  $ git checkout master
+  $ git fetch upstream master
+  $ git pull upstream master --ff-only
+
+
+:strong:`Create A Testing Branch`
```

Comment:
Why? Everything you list can easily be done in "detached HEAD" mode from `upstream/master`:
```
git checkout origin upstream/master
```
Having a `git worktree` at hand can also be useful to avoid interrupting current work.

Reply:
This is too advanced. Lets follow KISS here.